### PR TITLE
Add docs on newline chars in Cypher

### PR DESCRIPTION
**Pull request by gem-neo4j**

### CODE REVIEW

**Comment by gem-neo4j** [comment]
[GQL compliance - What are newline characters?](https://trello.com/c/a07Q9O90/939-gql-compliance-what-are-newline-characters)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-892.surge.sh

Updated pages:
- [Parsing](https://neo4j-docs-cypher-892.surge.sh/cypher-manual/5/syntax/parsing)

**Review by loveleif — APPROVED** [review]

File: modules/ROOT/pages/syntax/parsing.adoc [hunk 1]
```diff
@@ -54,4 +54,18 @@ The following example query uses vertical tabulation (`\u000B`) as whitespace be
...
+
+A newline character identifies a new line in the query and is also considered whitespace.
+The supported newline characters in Cypher are:
+
+[options="header", cols="1,2"]
+|===
+| Description                       | List of included Unicode Characters
+| Line Feed                         | `\n`, `\u000A`
+| Carriage Return                   | `\r`, `\u000D`
+| Carriage Return + Line Feed       | `\r\n`, `\u000D\u000A`
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/syntax/parsing.adoc]:
```suggestion
| Line feed                         | `\n`, `\u000A`
| Carriage return                   | `\r`, `\u000D`
| Carriage return + line feed       | `\r\n`, `\u000D\u000A`
```

There's more occurrences in the file where we don't adhere to sentence-style capitalization in table headers

File: modules/ROOT/pages/syntax/parsing.adoc [hunk 2]
```diff
@@ -54,4 +54,18 @@ The following example query uses vertical tabulation (`\u000B`) as whitespace be
...
+
+[[cypher-newlines]]
+== Supported newline characters
+
+A newline character identifies a new line in the query and is also considered whitespace.
+The supported newline characters in Cypher are:
+
+[options="header", cols="1,2"]
+|===
+| Description                       | List of included Unicode Characters
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/syntax/parsing.adoc]:
```suggestion
| Description                       | List of included Unicode characters
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-892.surge.sh

Updated pages:
- [Parsing](https://neo4j-docs-cypher-892.surge.sh/cypher-manual/5/syntax/parsing)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-892.surge.sh

Updated pages:
- [Parsing](https://neo4j-docs-cypher-892.surge.sh/cypher-manual/5/syntax/parsing)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by rsill-neo4j — APPROVED** [review]

**Pull request merged**